ginga/misc/plugins/TVMask.py

```diff
@@ -0,0 +1,447 @@
+"""Non-interactive masks display local plugin for Ginga."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from ginga.util.six import itervalues
+
+# STDLIB
+import re
+import os
+
+# THIRD-PARTY
+import numpy as np
+from astropy.io import fits
```

Comment:
@ejeschke , so what would be the best way to implement `fits.getdata(filename).astype(np.bool)` to satisfy such abstraction?